src/actions/Bluetooth/SensorUpdateActions.js

```diff
@@ -92,12 +95,24 @@ const startSetLogInterval = ({ macAddress, logInterval = 300 }) => async (dispat
   return result;
 };
 
-const updateSensor = sensor => async dispatch => {
+const updateSensor = sensor => async (dispatch, getState) => {
+  const state = getState();
```

Comment:
:this is a really minor thing.. I'd move the `getState()` to below the `logInterval` check - as it might not be needed. That would give a very small optimisation..? Also, the state is only used once, so no need to use a const..

```if (selectIsSyncingTemps(getState())) {```

Reply:
Nice, much tidier thanks 96a45bd